packages/core/src/modules/connections/services/ConnectionService.ts

```diff
@@ -257,7 +257,7 @@ export class ConnectionService {
    */
   public async processResponse(
     messageContext: InboundMessageContext<ConnectionResponseMessage>,
-    outOfBandRecord: OutOfBandRecord
+    invitationRecipientKeys: Key[]
```

Comment:
if we already set the `recipientKeyFingerprints` tag correctly, we don't have to pass the invitationRecipientkeys here, we can just extract them from the oob record